js_modules/dagster-ui/packages/ui-core/src/code-location/__fixtures__/CodeLocationPages.fixtures.ts

```diff
@@ -107,5 +113,40 @@ export const buildSampleOpsRootQuery = (config: {repoAddress: RepoAddress; opCou
       }),
     },
     delay: 2000,
+    maxUsageCount: 20,
+  });
+};
+
+export const buildSampleRepositoryGraphsQuery = (config: {
+  repoAddress: RepoAddress;
+  jobCount: number;
+  opCount: number;
+}) => {
+  const {repoAddress, jobCount, opCount} = config;
+  return buildQueryMock<WorkspaceGraphsQuery, WorkspaceGraphsQueryVariables>({
+    query: WORSKPACE_GRAPHS_QUERY,
+    variables: {
+      selector: repoAddressToSelector(repoAddress),
+    },
+    data: {
+      repositoryOrError: buildRepository({
+        usedSolids: new Array(opCount).fill(null).map(() => {
+          return buildUsedSolid({
+            definition: buildCompositeSolidDefinition({
+              name: faker.random.words(2).split(' ').join('-').toLowerCase(),
+            }),
+          });
+        }),
+        pipelines: new Array(jobCount).fill(null).map(() => {
+          return buildPipeline({
+            id: faker.datatype.uuid(),
+            graphName: faker.random.words(2).split(' ').join('-').toLowerCase(),
+            isJob: true,
+          });
+        }),
+      }),
+    },
+    delay: 2000,
+    maxUsageCount: 100,
```

Comment:
TIL

Reply:
Yep, me too. It turns out that otherwise, when you click back and forth between the sections in the storybook example, it repeats the query but fails because it hasn't been mocked more than once! 😆